### ISSUE TITLE
[v1.0.0-rc5] Failed to verify and cleanup any mangle table rule

### ISSUE THREAD

**Issue by christophlehmann** [cloudnativelabs/kube-router#932]
I updated to v1.0.0-rc5 and rebooted all cluster nodes. In the logs I see

```
Try `iptables -h' or 'iptables --help' for more information.
Failed to verify and cleanup any mangle table rule to FMWARD the traffic to external IP due to Failed to cleanup iptables command to set up FWMARK due to running [/sbin/iptables -t mangle -C PREROUTING -d 192.168.60.90 -m tcp -p tcp --dport 80 -j MARK --set-mark 7424 <nil> --wait]: exit status 2: iptables v1.8.3 (legacy): MARK: trailing garbage after value for option "--set-mark".
```

`<nil>` seems wrong to me

k8s version: 1.17.6
CNI Config:
```json
{
   "cniVersion":"0.3.0",
   "name":"kubernetes",
   "plugins":[
      {
         "name":"kubernetes",
         "type":"bridge",
         "bridge":"kube-bridge",
         "isDefaultGateway":true,
         "ipam":{
            "subnet": "10.222.64.0/21",
            "type":"host-local"
         }
      }
   ]
}
```

Content of the mangle table: (it's empty)

```
iptables --list PREROUTING -t mangle
Chain PREROUTING (policy ACCEPT)
target     prot opt source               destination
```

**Comment by hlubek** [comment]
This seems to be fixed by #928

**Comment by christophlehmann** [comment]
yes!
